### DISCUSSION TITLE
No Output Directory named "public" found

### DISCUSSION THREAD

**Discussion by SagnikPradhan** [vercel/vercel#4796] [Help]
![image](https://user-images.githubusercontent.com/26499576/86715271-add24c80-c03d-11ea-8e29-15294918c0e9.png)
![image](https://user-images.githubusercontent.com/26499576/86715340-c2164980-c03d-11ea-9bb4-5d709908a347.png)

What am I doing wrong? I did configure the settings properly yet `yarn dev --listen 5000` doesn't seem to work.

**Comment by paulogdm** [comment]
Are you able to share a minimal reproducible example?

**Comment by SagnikPradhan** [comment]
Eh. Not really. I ended up deleting the project itself. But if I remember
correctly, the project had a deeply nested output folder, I did set up the
same on the dashboard but it still errored. IG the CLI was not using the
directory I setup on dashboard but instead was looking for a "public"
folder in the root directory.
